iothub/device/src/IotHubDeviceClient.cs

```diff
@@ -116,35 +116,34 @@ public void SetConnectionStatusChangeHandler(Action<ConnectionInfo> statusChange
         /// <summary>
         /// Explicitly open the DeviceClient instance.
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
-        /// <exception cref="OperationCanceledException">Thrown when the operation has been canceled.</exception>
         /// </summary>
+        /// <exception cref="OperationCanceledException">Thrown when the operation has been canceled.</exception>
         public Task OpenAsync(CancellationToken cancellationToken = default) => InternalClient.OpenAsync(cancellationToken);
 
         /// <summary>
         /// Sends an event to a hub
         /// </summary>
+        /// <remarks>
```

Comment:
I try to follow the order of:
- summary
- remarks
- params
- returns
- exceptions
- example

Reply:
Good order to follow.